src/components/pages/utils/inntekt.ts

```diff
@@ -39,9 +39,38 @@ export const formatInntektToNumber = (s?: string | null | undefined) => {
 }
 
 export const updateAndFormatInntektFromInputField = (
```

Comment:
Har allerede gikk feedback på navnet på denne funksjonen. Den gjør ikke en update og en formatering, det er omvendt. Navnet er derfor misvisende.
forslag f.eks: formatAndUpdateBeloep

Videre, innspill på parameter for å øke lesbarhet/forståelse:
-   e: React.ChangeEvent<HTMLInputElement>,
-   previousFormattedBeloep: string,
-   inputValue: string,
-   updateBeloep / callback: (s: string) => void